src/Introspection/__EnumValue.hack

```diff
@@ -0,0 +1,19 @@
+namespace Slack\GraphQL\Introspection;
+
+use namespace Slack\GraphQL;
+
+<<GraphQL\InterfaceType('__EnumValue', 'Enum value introspection')>>
```

Comment:
These should be object types, not interfaces in GraphQL. If we're not allowing `<<GraphQL\ObjectType>>` on Hack interfaces, we might have to allow an exception for these introspection types.